motion_control/controllers/pose_pid_controller/include/pose_pid_controller/PosePIDControllerParameters.hpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2023 COGIP Robotics association <cogip35@gmail.com>
+// This file is subject to the terms and conditions of the GNU Lesser
+// General Public License v2.1. See the file LICENSE in the top level
+// directory for more details.
+
+/// @ingroup    pose_pid_controller Pose PID controller parameters
+/// @{
+/// @file
+/// @brief      Pose PID controller parameters
+/// @author     Eric Courtois <eric.courtois@gmail.com>
+/// @author     Gilles DOFFE <g.doffe@gmail.com>
+
+#pragma once
+
+// Project includes
+#include "pid_ng/PID.hpp"
+
+namespace cogip {
+
+namespace motion_control {
+
+/// @brief Position controller parameters
+class PosePIDControllerParameters {
+public:
+    /// @brief Constructor
+    /// @param pid  PID parameters
+    PosePIDControllerParameters(
+        cogip::pid::PID *pid = nullptr
```

Comment:
When you are in `cogip` namespace, you don't need to specify `cogip::`.

Reply:
done